semver.py

```diff
@@ -8,7 +8,7 @@
 import sys
 
 
-__version__ = "2.9.0"
+__version__ = "2.9.1-rc2"
 __author__ = "Kostiantyn Rybnikov"
 __author_email__ = "k-bx@k-bx.com"
 __maintainer__ = "Sebastien Celles"
```

Comment:
Maybe we should have a list here with you name also.

Reply:
Ok, created a list.